lib/src/vector_math/matrix2.dart

```diff
@@ -215,6 +215,8 @@ class Matrix2 {
   }
 
   /// Returns a new vector or matrix by multiplying this with [arg].
+  @pragma('wasm:prefer-inline')
+  @pragma('vm:prefer-inline')
   dynamic operator *(dynamic arg) {
     if (arg is double) {
       return scaled(arg);
```

Comment:
From Stephens comments it seems like dart2js won't inline functions with multiple return values. But we can change these functions to only have a single return, allowing dart2js to inline them, no?

```dart
dynamic result;
if (...) {
  result = ...;
} else if (...) {
  result = ...;
} else {
  throw ...
},
return result;
```
and add `@pragma('dart2js:prefer-inline')`

That may also improve the benchmarks using old API methods in dart2js similar to dart2wasm&vm

(also other places)